generators/language-python-flask/index.js

```diff
@@ -92,9 +93,21 @@ module.exports = class extends Generator {
 		//Devops
 		this.composeWith(require.resolve('../service-alert-notification'), {context: this.context});
 	}
-
+	//use this as a base to add dependencies to pipfile if not already there
 	_addDependencies(serviceDepdendenciesString){
 		let requirementsTxtPath = this.destinationPath(PATH_REQUIREMENTS_TXT);
+		let pipfilePath = this.destinationPath(PATH_PIPFILE);
```

Comment:
We need to make some modifications to append the Pipfile from each service to the correct section (`[packages]` and `[dev-packages]`). Also, serviceDependenciesString is only giving you the string from the 'requirements.txt` so we should also include the Pipfile. Check out  **line 27**  of this file and `lib/generatorbase` **line 99**.